package proto_ff;

import "yd_fieldoptions.proto";


message E_MarketMarket
{
	optional int32 m_itemId = 1[(yd_fieldoptions.field_cname) = "道具id"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "所属类型组"];
	optional int32 m_order = 3[(yd_fieldoptions.field_cname) = "排序"];
	optional int32 m_readyTime = 4[(yd_fieldoptions.field_cname) = "准备时间"];
	optional int32 m_sjTime = 5[(yd_fieldoptions.field_cname) = "上架时间"];
	optional int32 m_endTime = 6[(yd_fieldoptions.field_cname) = "竞拍时间"];
	optional int32 m_priceType = 7[(yd_fieldoptions.field_cname) = "交易货币"];
	optional int32 m_vieType = 8[(yd_fieldoptions.field_cname) = "是否支持竞价"];
	optional int32 m_startPrice = 9[(yd_fieldoptions.field_cname) = "竞拍底价"];
	optional int32 m_oncePrice = 10[(yd_fieldoptions.field_cname) = "每次增加的价格"];
	optional int32 m_directPrice = 11[(yd_fieldoptions.field_cname) = "一口价"];
	optional int32 m_gzType = 12[(yd_fieldoptions.field_cname) = "是否关注"];
	optional string m_gztuType = 13[(yd_fieldoptions.field_cname) = "关注获取途径", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_recoveryType = 14[(yd_fieldoptions.field_cname) = "是否系统回收"];
}


message Sheet_MarketMarket
{
	repeated E_MarketMarket E_MarketMarket_List = 1[(yd_fieldoptions.field_arysize)=1024];
}

message E_MarketBig
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "大类id"];
	optional string m_bigName = 2[(yd_fieldoptions.field_cname) = "大类名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_xb = 3[(yd_fieldoptions.field_cname) = "性别"];
}


message Sheet_MarketBig
{
	repeated E_MarketBig E_MarketBig_List = 1[(yd_fieldoptions.field_arysize)=16];
}

message E_MarketSmall
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "子页签id"];
	optional string m_smallName = 2[(yd_fieldoptions.field_cname) = "子页签名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_openLevel = 3[(yd_fieldoptions.field_cname) = "显示等级"];
	optional int32 m_bigid = 4[(yd_fieldoptions.field_cname) = "大类id"];
}


message Sheet_MarketSmall
{
	repeated E_MarketSmall E_MarketSmall_List = 1[(yd_fieldoptions.field_arysize)=128];
}

message E_MarketConstant
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	optional int32 m_taxRate = 2[(yd_fieldoptions.field_cname) = "出售税率"];
	optional int32 m_BPsellTime = 3[(yd_fieldoptions.field_cname) = "帮派拍品上架时间"];
	optional int32 m_sellNum = 4[(yd_fieldoptions.field_cname) = "上架数量"];
	optional int32 m_recycleNum = 5[(yd_fieldoptions.field_cname) = "回收数量"];
	optional int32 m_gz = 6[(yd_fieldoptions.field_cname) = "关注上限"];
	optional string m_EpackageType = 7[(yd_fieldoptions.field_cname) = "包裹类型", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_dh = 8[(yd_fieldoptions.field_cname) = "钻石兑换元宝"];
}


message Sheet_MarketConstant
{
	repeated E_MarketConstant E_MarketConstant_List = 1[(yd_fieldoptions.field_arysize)=2];
}
